src/sst/core/sync/threadSyncDirectSkip.cc

```diff
@@ -54,11 +54,11 @@ ThreadSyncDirectSkip::~ThreadSyncDirectSkip()
             CALL_INFO, 1, 0, "ThreadSyncDirectSkip total wait time: %lg seconds.\n", totalWaitTime);
 }
 
-ActivityQueue*
-ThreadSyncDirectSkip::getQueueForThread(int UNUSED(tid))
-{
-    return nullptr;
-}
+// ActivityQueue*
```

Comment:
Can delete this commented out function? Declaration is deleted from header.

Reply:
Yep, I usually catch those while review the diffs.